packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -0,0 +1,63 @@
+import API, { SSEHandler, SSEStatus } from '@/api/api';
+import type { TaskResponse } from '@/types/Types';
+import { TaskStatus } from '@/types/Types';
+import { logger } from '@/utils/logger';
+
+/**
+ * Fetches model card data from the server.
+ * @param {string} documentId - The document ID.
+ * @param {string} modelId - The model ID.
+ * @returns {Promise<TaskResponse|null>} A promise resolving to the task response or null if unsuccessful.
+ */
+export async function modelCard(documentId: string, modelId: string): Promise<TaskResponse | null> {
+	try {
+		const response = await API.post('/gollm/model_card', null, {
```

Comment:
We use `kebab-case` for URL, could you please update the back-end controller as well.
```suggestion
		const response = await API.post('/gollm/model-card', null, {
```